notifications-kibana-plugin/public/pages/Notifications/containers/Notifications.tsx

```diff
@@ -261,13 +141,13 @@ export default class Notifications extends Component<
         queryObject
       );
       const { notifications, totalNotifications } = getNotificationsResponse;
-      this.setState({ notifications, totalNotifications });
+      this.setState({ items: notifications, total: totalNotifications });
     } catch (err) {
       this.context.notifications.toasts.addDanger(
         getErrorMessage(err, 'There was a problem loading the managed indices')
```

Comment:
this is my mistake, could you correct the wording?

Reply:
updated